src/Altinn.Profile.Integrations/OrganizationNotificationAddressRegistry/Models/RegistryRequest.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System.Text.Json.Serialization;
+
+namespace Altinn.Profile.Integrations.OrganizationNotificationAddressRegistry.Models
+{
+    /// <summary>
+    /// Wrapper class for contact information to be sent to the registry.
+    /// This structure aligns with the external API requirements for the organization notification address registry.
+    /// </summary>
+    public record RegistryRequest
+    {
+        /// <summary>
+        /// Gets or sets Contact information
+        /// </summary>
+        [JsonPropertyName("Kontaktinformasjon")]
+        public ContactInfoModel? ContactInfo { get; set; }
+    }
+
+    /// <summary>
+    /// External model for contact info
+    /// </summary>
+    public record ContactInfoModel
+    {
+        /// <summary>
+        /// ConfirmedDate
+        /// </summary>
+        [JsonPropertyName("bekreftetDato")]
+        public string? ConfirmedDate { get; set; }
+
+        /// <summary>
+        /// Gets or sets the object containing alert info
```

Comment:
Nitpicking: replace _Gets or sets_ "fillers" with something of descriptive value
```suggestion
        /// A channel for digital contact, either email or phone
```